web-ng/src/app/components/layer-dialog/layer-dialog.component.ts

```diff
@@ -175,7 +177,7 @@ export class LayerDialogComponent implements OnDestroy {
       let field: Field = {
         id: fieldId,
         type: FieldType['TEXT'],
-        required: false,
+        required: question.required || false,
```

Comment:
Can this ever actually be undefined or null?

Reply:
Yes this cannot be undefined as it has default value false